src/app/app.component.scss

```
@@ -31,3 +31,28 @@ hr {
     margin-bottom: 30px;
     width: 100%;
 }
+
+.source_label mat-icon {
+    font-size: 25px;
+    padding-right: 10px;
+}
+
+mat-icon.color_success { 
+    color: #4CAF50;
```

_Nice to have:_

Ich bin Fan davon, sowas wie Farben global verfügbar zu haben, damit man nicht immer die gleichen Farben kopiert, die man hinterher vielleicht auch nochmal anpassen muss/möchte.
Das kann man relativ easy über CSS-Custom-Properties machen.

Dazu muss man zur Datei `styles.css` den folgenden Block hinzufügen:

```scss
:root {
    --success-color: #4CAF50;
}
```

Damit legst du global unter der Variable `--sucess-color` diese Farbe fest.
Verwenden kannst du es dann einfach so:

```suggestion
    color: var(--success-color);
```